extensions/sql-database-projects/src/dialogs/publishDatabaseDialog.ts

```diff
@@ -122,7 +122,7 @@ export class PublishDatabaseDialog {
 						title: constants.targetDatabaseSettings,
 						components: [
 							{
-								title: constants.profileWarningText,
+								title: '',
```

Comment:
we still need the empty string? cant remove it?

Reply:
nope FormComponents need a title :(